### PR TITLE
Zina/python/25 may2021

### PR DESCRIPTION
**Pull request by ZinaYeghiazaryan**
Pascal.py file prints out the first n rows of Pascal's triangle. 
Perfect_Number.py file checks whether a number is perfect or not.
Fibonacci.py file prints out the n-th Fibonacci item.

### CODE REVIEW

File: Zina_Yeghiazaryan/python/25_05_2021/Fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        return 1
+    return Fib(n - 1) + Fib(n - 2)
+
+while True:
+    try:
+        n = int(input("Enter the number: "))
+        print(str(n) + " Fibonacci item is: " + str(Fib(n - 1)))
+    except ValueError:
+        print("Entered wrong value. Please enter a number.")
+        print()
```
Comment (ani-im) [inline Zina_Yeghiazaryan/python/25_05_2021/Fibonacci.py]: for 0 value 
image

for 1000
image
 
 for -2
 
image

Comment (ani-im) [reply Zina_Yeghiazaryan/python/25_05_2021/Fibonacci.py]: Correct style for function_name, file name based on pep8

File: Zina_Yeghiazaryan/python/25_05_2021/Pascal.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        print("\n")
+
+while True:
+    try:
+        n = int(input("Enter the number of row: "))
+        print("First " + str(n) + " rows of Pascal's triangle: ")
+        Pascal(n)
+    except ValueError:
+        print("Entered wrong value. Please enter a number.")
+        print()
```
Comment (ani-im) [inline Zina_Yeghiazaryan/python/25_05_2021/Pascal.py]: 1. why double?
image

2. Fix for big number (e.g 1000000)
image

Comment (ani-im) [reply Zina_Yeghiazaryan/python/25_05_2021/Pascal.py]: again fix styles based on pep8

File: Zina_Yeghiazaryan/python/25_05_2021/Perfect_Number.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    else:
+        print(str(n) + " is not a perfect number.")
+
+while True:
+    try:
+        n = int(input("Enter the number: "))
+        Check(n)
+    except ValueError:
+        print("Entered wrong value. Please enter a number.")
+        print()
```
Comment (ani-im) [inline Zina_Yeghiazaryan/python/25_05_2021/Perfect_Number.py]: Perfect number should be positive number
- https://www.britannica.com/science/perfect-number

Coding style with pep8

**Review by ani-im — CHANGES_REQUESTED** [review]

**Comment by ZinaYeghiazaryan** [comment]
https://github.com/Artur-Aghajanyan/VSU_ITC/tree/Zina/python/25May2021/Zina_Yeghiazaryan/python/25_05_2021

**Pull request merged**